requirements.txt

```diff
@@ -7,5 +7,6 @@ pyOpenSSL==17.5.0
 # We need to disable ruamel.yaml for now because of test failures
 #ruamel.yaml
 six==1.10.0
+cmd2==0.8.7
```

Comment:
What's this required for?

Reply:
I'll break this out in a separate PR - `shade` dependency `cmd2` 0.9.0 now required py3 - see https://travis-ci.org/openshift/openshift-ansible/jobs/385637999

Reply:
sorry, i should've looked at the commit logs too, i'm fine with leaving this in the PR as it's a separate commit

Reply:
Created https://github.com/openshift/openshift-ansible/pull/8571